.github/workflows/ci.yml

```diff
@@ -95,6 +95,8 @@ jobs:
       - name: Install alsa and udev
         run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev libwayland-dev libxkbcommon-dev
       - name: CI job
+        # To run the tests one item at a time for troubleshooting, use
+        # cargo --quiet test --lib -- --list | sed 's/: test$//' | MIRIFLAGS="-Zmiri-disable-isolation" xargs -n1 cargo miri test --lib -- --exact
```

Comment:
I think this needs to also specify `-Zmiri-permissive-provenance -Zmiri-disable-weak-memory-emulation` otherwise some tests wont pass

Reply:
```suggestion
        # cargo --quiet test --lib -- --list | sed 's/: test$//' | MIRIFLAGS="-Zmiri-disable-isolation" xargs -n1 cargo miri test --lib -- --exact -Zmiri-permissive-provenance -Zmiri-disable-weak-memory-emulation
```